### DISCUSSION TITLE
Подскажите, где на телефоне должен лежать файл list.txt, который подгружается командой --host в командной строке?

### DISCUSSION THREAD

**Discussion by babitsyn** [dovecoteescapee/ByeDPIAndroid#170] [Q&A]
Подскажите, где на телефоне должен лежать файл list.txt, который подгружается командой --host в командной строке?

**Comment by dDmioko** [comment]
Видимо пока поддержки файлов нет [issues/75](issues/75)\
Пробовал через относительный путь - не помогает. Внутрь data папки вставить файл мне ОС блочит
